src/Mapbender/CoreBundle/Resources/public/mapbender.element.gpsPosition.js

```diff
@@ -134,11 +134,22 @@
         _centerMap: function (point) {
             var olmap = this.map.map.olMap,
                 extent = olmap.getExtent();
-            if (extent.containsLonLat(point) === false || true === this.options.follow) {
-                olmap.panTo(point);
-            } else if (this.firstPosition && this.options.centerOnFirstPosition) {
-                olmap.panTo(point);
+            if (extent.containsLonLat(point) === false) // point is in extent?
+            {
+                if (this.options.follow) {
+                    console.debug("this.options.follow");
```

Comment:
Please use `console.log` instead of `console.debug` to be more IE friendly. More info here: http://caniuse.com/#feat=console-basic